### PR TITLE
add solid icons

### LINKED ISSUE #138: Add a icon for solidjs
https://www.solidjs.com/

https://www.solidjs.com/media

See https://github.com/getsentry/sentry-javascript/discussions/5552

### PR DESCRIPTION
**Pull request by atilafassina**
fix #138 

gently summoning @AbhiPrasad 🙏

### CODE REVIEW

File: svg/solid.svg [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Jesse-Box) [inline svg/solid.svg:1]: Remove the following attributes from line 1

`width`
`height`
`fill`

File: svg/solid.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<rect width="20" height="20" fill="white"/>
```
Comment (Jesse-Box) [inline svg/solid.svg:2]: Modify the following attribute values from line 2
`width="100%"`
`height="100%"`

File: svg_80x80/solid.svg [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Jesse-Box) [inline svg_80x80/solid.svg:1]: Remove the following attributes from line 1

`width`
`height`
`fill`

File: svg_80x80/solid.svg [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+<g clip-path="url(#clip0_1_10)">
```
Comment (Jesse-Box) [inline svg_80x80/solid.svg:2]: Include the following code above line 2 (The same as the smaller icon)
`<rect width="100%" height="100%" fill="white"/>`

**Review by Jesse-Box — CHANGES_REQUESTED** [review]
Hey @atilafassina, Just a few small changes before this can be merged :)

**Comment by atilafassina** [comment]
> Hey @atilafassina, Just a few small changes before this can be merged :)

Thank you for the quick reply, @Jesse-Box 
all changes addressed.

looking at the preview, they're now kind of the same image.
(8080 has the opaque white background instead of transparent, and the small has no size constraint - so let me know if I did anything wrong)

**Pull request merged**